src/SLIP10Node.test.ts

```diff
@@ -289,6 +290,28 @@ describe('SLIP10Node', () => {
       });
     });
 
+    it('initializes a new node from a derivation path with a Uint8Array', async () => {
+      const node = await SLIP10Node.fromDerivationPath({
+        derivationPath: [
+          defaultBip39BytesToken,
+          BIP44PurposeNodeToken,
+          `bip32:60'`,
+        ],
+        curve: 'secp256k1',
+      });
+
+      const stringNode = await SLIP10Node.fromDerivationPath({
+        derivationPath: [
+          defaultBip39NodeToken,
+          BIP44PurposeNodeToken,
+          `bip32:60'`,
+        ],
+        curve: 'secp256k1',
```

Comment:
Should we test it for both curves?

Reply:
Curve is a BIP-32 thing, not BIP-39. I can test it if you prefer, but I don't think it's necessary.

Reply:
Fair enough 👍